pkg/config/remote/api/http.go

```diff
@@ -42,6 +47,7 @@ func NewHTTPClient(baseURL, apiKey, appKey string) *HTTPClient {
 		Transport: httputils.CreateHTTPTransport(),
 	}
 
+	baseURL := config.GetMainEndpoint("https://config.", "remote_configuration.rc_dd_url")
```

Comment:
what about using `net/url` parse so that you can also use url.Join to append paths